File: Iniciante/1012 - Área/JavaScript/1012.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split(" ");
  const [a, b, c] = lines.map(Number);
  const PI = 3.14159;
  const triangleArea = calculateTriangleArea(a, c);
  const circleArea = calculateCircleArea(c, PI);
  const trapezoidArea = calculateTrapezoidArea(a, b, c);
  const squareArea = calculateSquareArea(b);
  const rectangleArea = calculateRectangleArea(a, b);
  console.log(`TRIANGULO: ${triangleArea.toFixed(3)}`);
  console.log(`CIRCULO: ${circleArea.toFixed(3)}`);
  console.log(`TRAPEZIO: ${trapezoidArea.toFixed(3)}`);
  console.log(`QUADRADO: ${squareArea.toFixed(3)}`);
  console.log(`RETANGULO: ${rectangleArea.toFixed(3)}`);
}

function calculateTriangleArea(cathetus1, cathetus2) {
  return (cathetus1 * cathetus2) / 2.0;
}

function calculateCircleArea(radius, pi) {
  return pi * radius ** 2;
}

function calculateTrapezoidArea(base1, base2, height) {
  return ((base1 + base2) * height) / 2.0;
}

function calculateSquareArea(side) {
  return side * side;
}

function calculateRectangleArea(base, height) {
  return base * height;
}

main();

File: Iniciante/1012 - Área/Python/1012.py
def main():
    a, b, c = map(float, input().split())
    PI = 3.14159
    triangle_area = calculate_triangle_area(a, c)
    circle_area = calculate_circle_area(c, PI)
    trapezoid_area = calculate_trapezoid_area(a, b, c)
    square_area = calculate_square_area(b)
    rectangle_area = calculate_rectangle_area(a, b)
    print(f"TRIANGULO: {triangle_area:.3f}")
    print(f"CIRCULO: {circle_area:.3f}")
    print(f"TRAPEZIO: {trapezoid_area:.3f}")
    print(f"QUADRADO: {square_area:.3f}")
    print(f"RETANGULO: {rectangle_area:.3f}")


def calculate_triangle_area(cathetus1, cathetus2):
    return (cathetus1 * cathetus2) / 2.0


def calculate_circle_area(radius, pi):
    return pi * (radius**2)


def calculate_trapezoid_area(base1, base2, height):
    return ((base1 + base2) * height) / 2.0


def calculate_square_area(side):
    return side * side


def calculate_rectangle_area(base, height):
    return base * height


if __name__ == "__main__":
    main()
